docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -497,6 +503,41 @@ docker/gremlin-server.sh 3.4.2
 To be a bit more clear, the version can not be a Docker tag like "latest" because there is no such TinkerPop artifact
 that has been published with that version number.
 
+[[docker-testing]]
+== Testing with Docker
+
+Some GLV allow you to run tests without Gremlin Server build. Currently supported gremlin-go, gremlin-javascript and
+gremlin-dotnet.
+
+Please make sure Docker is installed and running on your system. 
+You will need to install both link:https://docs.docker.com/engine/install/[Docker Engine] and link:https://docs.docker.com/compose/install/[Docker Compose], which are included in link:https://docs.docker.com/desktop/[Docker Desktop].
+
+The docker compose environment variable `GREMLIN_SERVER` specifies the Gremlin server docker image to use, i.e. an
```

Comment:
Can we have it as optional? If it is not specified default image will be pulled?

Reply:
default version implemented only in `gremlin-go/run.sh` as I understand